packages/scene-composer/src/components/Tree/tree.scss

```diff
@@ -25,145 +25,121 @@ $active-color: #ec7211;
   color: $active-color;
 }
 
-  #tm-main {
-    margin-bottom: 5px;
-
-    .tm-tree {
-      border-top: $border;
-      list-style-position: outside;
-      list-style-type: none;
-      margin: 0;
-      padding: 0;
-      position: relative;
-      user-select: none;
-
-      &.drop,
-      .drop {
-        border-bottom: 3px solid #fff;
-      }
-
-      .tm-tree-item {
-        padding-left: $indent * 2;
-        border-top: 1px solid $background-hover;
+#tm-main {
+  margin-bottom: 5px;
+
+  .tm-tree {
+    border-top: $border;
+    list-style-position: outside;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    position: relative;
+    user-select: none;
+
+    &.drop,
+    .drop {
+      border-bottom: 3px solid #fff;
+    }
 
-        &::marker {
-          padding: 0;
-          margin: $indent * -1;
-          display: inline-block;
+    .tm-tree-item {
+      padding-left: $indent * 2;
+      border-top: 1px solid $background-hover;
+
+      .tm-tree-item-inner {
+        cursor: pointer;
+        margin-left: -2rem;
+        padding-left: 2rem;
+        align-items: center;
+        display: flex;
+
+        input[type="radio"] {
+          margin-right: 1rem;
+
+          // styled to match Polaris: https://polaris.a2z.com/components/radio-group/
+          &:checked {
+            appearance: none;
+            background-color: #fff;
+            color: #fff;
+            border: 4px solid #00a1c9;
+            border-radius: 50%;
+            padding: 3px;
+          }
```

Comment:
These should be pulling styles from Polaris directly, as opposed to being hard coded here. I'm ok with shipping this, but we should spend a bit of time seeing if we can use their CSS variables here instead.

Reply:
We can create our own component. Unfortunately the Polaris radio button doesn't allow for passing unique identifiers so React throws the "Each child should have a unique key prop" error.